src/f_finale.c

```diff
@@ -38,6 +38,7 @@
 #include "d_deh.h"  // Ty 03/22/98 - externalizations
 #include "m_misc2.h" // [FG] M_StringDuplicate()
 #include "m_swap.h"
+#include "r_draw.h"
```

Comment:
Do we still need to `#include m_menu.h` here?

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/778/commits/6ed1d5d7a2401c698c64da2015c0d837d7f8b4fa